core/src/main/scala/com/avast/dapper/CqlType.scala

```diff
@@ -0,0 +1,41 @@
+package com.avast.dapper
+
+sealed trait CqlType
+
+object CqlType {
+
+  // all following types are case class with no param intentionally - it't used like a better enum both into Scala and Java, case object cannot be used
```

Comment:
Didn't we agree that sharing things between Java and Scala API might not be that good? Because this pattern is really strange... usually you expect to have a single instance of enum value, this allows you to create many.

Reply:
It's not for Java API; it's about using these types in annotations. If they were `case objects`s, you would have to write `CqlType.Int.type` which is ugly..

Reply:
I think you could make some type aliases inside `CqlType` companion object to have both at the same time.